addons/mail/static/src/core/web/chatter.xml

```diff
@@ -93,7 +93,12 @@
             </div>
             <t t-if="state.thread.composer.type">
                 <t t-if="state.thread.composer.type === 'message'">
-                    <div class="flex-shrink-0 px-3 pt-3 text-truncate small" style="margin-left:48px" t-out="toFollowersText"/>
+                    <div class="flex-shrink-0 px-3 pt-3 text-truncate small" style="margin-left:48px">
+                        <t t-out="toFollowersText"/>
+                        <button class="btn btn-link" t-on-click="onClickFollowerList" title="Show all recipients">
+                            <i class="fa fa-caret-down"/>
+                        </button>
```

Comment:
Should not display Follower List button if there is no followers yet:
![Screenshot 2023-06-27 at 18 25 37](https://github.com/odoo/odoo/assets/6569390/ff601906-245e-4b72-a501-4584781aea8d)

Reply:
Should we remove the while listing UI including the `to:`. It make the template simple but could confuse the user when there is absolutely nothing?

Reply:
Better not showing anything, including no _"To: "_. No reason to show the recipients if there are none :)

Reply:
I do agree to remove the display followers button when there are none, because it does not make sense.... 
I'll keep the 'To:" however, it emphasizes that the list is empty IMO.